src/test/resources-src/multi-jdk/src/main/java/nl/basjes/maven/multijdk/AbstractJavaVersion.java

```diff
@@ -0,0 +1,32 @@
+package nl.basjes.maven.multijdk;
```

Comment:
unless your heart is set on this I would rather keep code in the repo under the original `org.vafer` package. Or is there a particular reason you want `nl.basjes`?

Reply:
No, this can be changed.

Reply:
Can you update this?

Reply:
Yes, will update. Had a busy week.

Reply:
Updated